app/src/androidTest/java/com/duckduckgo/app/cta/ui/CtaViewModelTest.kt

```diff
@@ -278,15 +283,15 @@ class CtaViewModelTest {
 
     @Test
     fun whenCtaDismissedAndUserHasPendingOnboardingCtasThenStageNotCompleted() = runTest {
-        givenOnboardingActive()
+        givenDaxOnboardingActive()
```

Comment:
This was changed everywhere as we had 2 identical functions `givenOnboardingActive()` / `givenDaxOnboardingActive()`. Removed one of them.